src/components/pages/pages.pw.tsx

```diff
@@ -161,9 +161,8 @@ test.describe("Accessibility tests for Pages component", () => {
       await mount(<PagesComponent initialpageIndex={propValue} />);
 
       await dataComponentButtonByText(page, "Open Preview").click();
-      const pageComponentTitle = getDataElementByValue(page, "title").nth(1);
 
-      await checkAccessibility(page, pageComponentTitle);
```

Comment:
suggestion(non-blocking): I had to add these `pageComponentTitle` to avoid some flakiness I witnessed when doing the refactor. The accessibility checks were running before the component had loaded.

Reply:
Hmm, I removed these only because these tests started failing (I'd raised the PR initially without checking PW locally and they failed on here, and then failed again locally when I ran them), and it turned out removing that second paramater made them pass (and several times in a row, so I hoped it wasn't just flakiness). I didn't dig too deeply but I think I assumed my CSS changes had altered how the animations were happening enough that waiting for that element, which was necessary before, was now somehow breaking things.

I'll take a closer look locally at what is happening in the browser, both with and without that removal, to see if it's OK to remove - and if not I'll see if there's another solution to whatever was making it fail.

Reply:
Thanks for making me look at this again. From watching the replay, the accessibility tests in my version definitely don't seem to be running at the right time - it seems to be after PW has already navigated to a new page for some reason.

I've looked more deeply at why the tests were failing with these additional waits in, and believe I've come up with a better solution for those (for this one it's rewriting the `pageComponentTitle` locator, for the others it's changing a trivial bit of CSS in the component). Will get those changes pushed this afternoon 👍